tests/unit/containers/quiz/TutorAnswerGrading.spec.jsx

```diff
@@ -0,0 +1,142 @@
+import { screen, fireEvent, waitFor } from '@testing-library/react'
+import { describe, it, expect, vi, beforeEach, afterEach } from 'vitest'
```

Comment:
```suggestion
import { vi } from 'vitest'
```
I think it is would work

Reply:
done